File: mario.c
#include <cs50.h>
#include<stdio.h>

int main(void)
{
    int n,i,j,k;
    
    do
    {
        printf("Enter a non-negetive integer number no greater than 23 : ");
        n = get_int();
    }
    while((n>23)||(n<0));
    for(i = 1;i <= n;i++)
    {
        for(j = 1;j <= n-i;j++)
        printf(" ");
        for(k = 1;k <= i+1;k++)
        printf("#");
        printf("\n");
    }
}

File: mario.py
import cs50

def main():
    while True:
        print("Height: ",end = "")
        n = cs50.get_int()
        if(n<=23 and n>=0):
            break
    for i in range (1,n+1):
        for j in range (1,n-i+1):
            print(" ",end = "")
        for k in range (1,i+1):
            print("#",end = "")
        print("#")
if __name__ == "__main__":
    main()
